### PR TITLE
Mechanical moves to make environment packages

### PR DESCRIPTION
**Pull request by slundqui**

### CODE REVIEW

File: policy_trainers/multi_agent_rllib.py [hunk 1]
```diff
@@ -4,23 +4,20 @@
...
-from ray.rllib.utils.test_utils import (
-    add_rllib_example_script_args,
-    run_rllib_example_script_experiment,
-)
+from ray.rllib.utils.test_utils import add_rllib_example_script_args, run_rllib_example_script_experiment
 from ray.tune.logger import pretty_print
 from ray.tune.registry import get_trainable_cls
 
+# Import registers hyperdrive envs
+# from agent0.traiderdaive import FullHyperdriveEnv
```
Comment (dpaiton) [inline policy_trainers/multi_agent_rllib.py]: dead comment?

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # This is doing an editable install on the relative path
+# TODO move these into pyproject.toml as requirements
+# This will require not doing a local install of agent0
+-e .
 -e ../agent0[all]
```
Comment (dpaiton) [inline requirements.txt:5]: this is weird, but I guess it's fine. Can you update README to include git clone agent0 in the parent folder?

**Review by dpaiton — APPROVED** [review]

File: policy_trainers/multi_agent_rllib.py (hunk 1 above)
Comment (slundqui) [reply policy_trainers/multi_agent_rllib.py]: yeah this was in the previous file, isort moved this chunk. I'll remove this comment

**Comment by wshainin** [comment]
LGTM

**Pull request merged**